apps/workflows/jobs/refill-daily.ts

```diff
@@ -29,14 +33,32 @@ client.defineJob({
             gt(table.refillAmount, table.remaining),
             or(
               isNull(table.lastRefillAt),
-              lte(table.lastRefillAt, t), // Check if more than 24 hours have passed
+              lte(table.lastRefillAt, tDay), // Check if more than 24 hours have passed
```

Comment:
this might backfire, consider this scenario

day 1: our cron job runs at 00:00:02 due to some delay, it sets the `lastRefillAt` to `day1 00:00:02`
day2: our cron job runs on time at 00:00:00 and doesn't update these keys now because it hasn't been exactly 24h

Reply:
No longer valid?